ProQuest.js

```diff
@@ -155,44 +160,53 @@ function getSearchResults(doc, checkOnly) {
 	return found ? items : false;
 }
 
+var map = {
+	"newspaperArticle" : "newspaperArticle",
+	"jnlArticle" : "journalArticle", //also magazineArticles
+	"dissertationPublished" : "thesis",
+	"reportSerial" : "report",
+	"workingPaper" : "report",
+	"book" : "book",
+	"webPage" : "webpage",
+	"blog" : "blogPost",
+	"report" : "report",
+	"audioVideo" : "videoRecording",
+	"default" : "conferencePaper" //also default = manuscript?, referenceWork, encyclopedia
+}
+
 function detectWeb(doc, url) {
 	initLang(doc, url);
-
-	followLink = false;
-
+	
+	followLink = true;
+	var active = doc.getElementsByClassName('active');
+	for (var i=0; i<active.length; i++) {
+		if ( active[i].id.indexOf("tab-AbstractRecord-null") > -1) {
+			followLink = false;
+		}
+	}
+	
 	//Check for multiple first
-	if (url.indexOf('docview') == -1 &&
-		url.indexOf('pagepdf') == -1) {
+	if (url.indexOf('docview') == -1 &&	url.indexOf('pagepdf') == -1) {
```

Comment:
nit: spacing after &&
Also for readability, could you wrap the if statement below (not your code, I realize) in {}?